app/adapters/redis.py

```diff
@@ -8,14 +8,17 @@ class Redis(_Redis):
 
 def dsn(
     scheme: str,
+    username: str | None,
```

Comment:
todo try something more like this
```suggestion
    username: str = "default",
```